cosmos/dbt/selector.py

```diff
@@ -295,7 +295,14 @@ def _should_include_node(self, node_id: str, node: DbtNode) -> bool:
         self.visited_nodes.add(node_id)
 
         if node.resource_type == DbtResourceType.TEST:
-            node.tags = getattr(self.nodes.get(node.depends_on[0]), "tags", [])
+            dependency_ids = [node_id for node_id in node.depends_on if node_id.startswith("model.")]
```

Comment:
Why do we only care about model dependencies? Why don't we care about potential parent seeds/sources tags?

Reply:
This is a good point and should be taken out